packages/config-plugins/src/ios/ProvisioningProfile.ts

```diff
@@ -33,21 +33,30 @@ export function setProvisioningProfileForPbxproj(
     : findFirstNativeTarget(project);
   const [nativeTargetId, nativeTarget] = nativeTargetEntry;
 
+  const quotedAppleTeamId = ensureQuotes(appleTeamId);
```

Comment:
seems like the inclusion of these files are from some weird main/master rebase issue. reverting...